interview-questions.md

```diff
@@ -8,50 +10,128 @@ Then, research the question to expand on your answer. Even if you feel you have
+
+                    GitHub offers free and paid products for storing and collaborating on code. Some products apply only to personal accounts, while other plans apply only to organization and enterprise accounts. For more information about accounts, see "Types of GitHub accounts."
+
+                    With GitHub Free for personal accounts, you can work with unlimited collaborators on unlimited public repositories with a full feature set, and on unlimited private repositories with a limited feature set.
 
-Researched answer:
+    Sited: https://docs.github.com/
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+    Your answer: Javascript operators that comes to mind are the && and === operators that return one result. Boolean value one result. 
+
+Researched answer:--> Boolean expressions in JavaScript return true or false. Even though true and false are words they are not strings.  
+                    They represent the value of true/false.
+
+                    Boolean values are very useful when making evaluations about data. Equality operators compare one value to the other and judge if the values equate to the same thing. If they do equate to the same a Boolean value of true is returned if not we get a Boolean value of false. There are two types of equality operators, the loose equality operator and the strict equality operator. The loose equality operator converts the two items to the same data type before making a comparison and the strict equality operator requires the two items being compared must be the same data type to return true.
+
+Sited: LEARN Academy, Golf Cohort-22', Syllabus, specifically JavaScript Introduction
 
-Researched answer:
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: An index, is zero-based, represented in arrays. Each index has an value. Still understanding the link between value and index.
+
+Researched answer:--> Value
+                    Each element in the array must be a JavaScript data type. The content of each element is called the value.
+
+                      Index
+                    Every value in an array has a particular location known as the index. Indexes are sequential numbers that are a bit like an address for every element in the array. Arrays are zero indexed which means the elements are numbered starting with the number 0 and increase by one whole number for each new element.
+
+                      Accessing Elements
+                    One of the many benefits to storing data in arrays is that we can access the individual values. Since the indexes are like an address, we can reference the variable that holds the entire array and the specific index in square brackets in order to retrieve the value. If the index does not exist the output will be undefined.
 
-Researched answer:
+Sited: LEARN Academy, Golf Cohort-22', Syllabus, specifically JavaScript Arrays
```

Comment:
While I appreciate that you are citing your sources, please make sure that you are not copying directly from these sources.  I want to see your understanding of the concepts from your research, not Google or the syllabus wording.